lua/extn/sky/unstable/engine/polysub.lua

```diff
@@ -5,7 +5,7 @@
 -- mutate global used by matron to select engine
 engine.name = "PolySub"
 
-local glue = include('we/lib/polysub')
+local glue = include('polysub')
```

Comment:
does `include` work here? i thought include would only work for files coming from `dust`. iirc everything coming out of the core needs `require` (except for sky stuff which has its own `use` system)